Source/Android/jni/MainAndroid.cpp

```diff
@@ -602,6 +605,75 @@ JNIEXPORT void JNICALL Java_org_dolphinemu_dolphinemu_NativeLibrary_eglBindAPI(J
   eglBindAPI(api);
 }
 
+JNIEXPORT void JNICALL Java_org_dolphinemu_dolphinemu_NativeLibrary_InitGameIni(
+JNIEnv* env, jobject obj, jobject assetManager, jstring jGameID)
+{
+  IniFile ini;
```

Comment:
As far as I can tell, this should be able to be moved below `mgr`, closer to its usage point.